site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -70,9 +70,29 @@ The specified Lambda Id must be of type `SAMLv2Reconcile`.
 
 include::_identity-provider-linking-strategy-request-parameter.adoc[]
 
+[field]#identityProvider.loginHintConfiguration.enabled# [type]#[boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.43.0#::
+When enabled and HTTP-Redirect binding are used, a login hint will be passed (when available) as a query parameter to the IdP using the configured parameter name set by the [field]#identityProvider.loginHintConfiguration.parameterName#.
+
+[field]#identityProvider.loginHintConfiguration.parameterName# [type]#[String]#::
+The name of the query parameter to use when sending a login hint to the IdP. This is only used when a login hint is available and [field]#identityProvider.loginHintConfiguration.enabled# is set to `true`.
+
 [field]#identityProvider.name# [type]#[String]# [required]#Required#::
 The name of this SAML v2 identity provider. This is only used for display purposes. Must be unique across all identity providers.
 
+ifndef::samlv2_idp_initiated[]
+[field]#identityProvider.nameIdFormat# [type]#[String]# [required]#Required# [default]#Defaults to `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`#::
```

Comment:
depending upon how this renders, you may want to move this to the description area - we have other examples of this for reference.